webapp/src/features/UserItem/UserItem.js

```diff
@@ -50,7 +54,7 @@ const UserItem = () => {
 					<BillingHistory />
 				</Col>
 				<Col lg={8}>
-					<ActivityHistory />
+					<ActivityHistory isGroupSubscription={false} />
```

Comment:
just for information
not sending isGroupSubscription from here will take its value as false